test/test_parser.cpp

```diff
@@ -1880,8 +1882,7 @@ TEST(Parser_BacklinkCount)
 
     // backlink count compared to link count
     verify_query(test_context, items, "@links.@count >= self.@count", 5);
-    verify_query(test_context, t, "items.@count >= items.@links.@count", 1);
```

Comment:
I think it could still be valuable to test over a single link and over a list, and they should functionally give the same result, right?